### PR TITLE
Remove passing receipt email to Stripe

### LINKED ISSUE #6543: We’ll need to stop setting the receipt_email field while creating a new payment intent.
pdjVC1-sB-p2#comment-672

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #6543

#### Description
As we have the plan to stop using Stripe's email generation mechanism, this PR removes setting this as part of the payment intent. More details - pdjVC1-sB-p2#comment-672

#### Testing instructions
Try to collect payment. Check receipt generation and sending

### CODE REVIEW

File: Hardware/Hardware/CardReader/PaymentIntentParameters.swift [hunk 1]
```diff
@@ -24,10 +24,6 @@ public struct PaymentIntentParameters {
     @StatementDescriptor
     public private(set) var statementDescription: String?
 
-    /// Email address that the receipt for the resulting payment will be sent to.
-    @Email
```
Comment (ctarda) [inline Hardware/Hardware/CardReader/PaymentIntentParameters.swift:28]: If you remove this property, it should also be safe to remove a struct called Email in the hardware module, which is where the `@Email` annotation is declared (this is called a [property wrapper](https://docs.swift.org/swift-book/LanguageGuide/Properties.html)):
`public struct Email<Value: StringProtocol>`

**Review by ctarda — COMMENTED** [review]
It is looking good. There is a property wrapper that could be removed as well, as it does not seem to be used anywhere else.

File: Hardware/Hardware/CardReader/PaymentIntentParameters.swift (hunk 1 above)
Comment (kidinov) [reply Hardware/Hardware/CardReader/PaymentIntentParameters.swift:28]: Good catch! Done - f48064cb64770dd5f5a904c6d393bab10789a663

**Review by jaclync — APPROVED** [review]
Code and testing on US & CA stores LGTM! :shipit:

**Pull request merged**